src/sync/init.ts

```diff
@@ -136,8 +150,12 @@ export async function init(session: ISession, opts: Options) {
 
   session.log.info(await FINISHED(session));
 
-  const { start } = await inquirer.prompt([questions.start()]);
-  if (!start) {
+  let start = false;
+  if (!opts.yes) {
+    const promptStart = await inquirer.prompt([questions.start()]);
+    start = promptStart.start;
+  }
+  if (!start && !opts.yes) {
```

Comment:
```suggestion
  if (!start) {
```
I think this is all you need since `false` is the default, and it keeps `opts.yes` only applying to questions/output.

Reply:
so "if we are not in `yes` mode, and user has decided not to `start`, send advisory message`

Reply:
Ah I see 👍